packages/container/tests/container_test.js

```diff
@@ -465,15 +481,19 @@ QUnit.test('factory for non extendables resolves are cached', function() {
   };
 
   deepEqual(resolveWasCalled, []);
-  container.lookupFactory('foo:post');
+  lookupFactory('foo:post', container);
   deepEqual(resolveWasCalled, ['foo:post']);
 
-  container.lookupFactory('foo:post');
+  lookupFactory('foo:post', container);
   deepEqual(resolveWasCalled, ['foo:post']);
 });
 
 QUnit.test('The `_onLookup` hook is called on factories when looked up the first time', function() {
-  expect(2);
+  if (isFeatureEnabled('ember-factory-for') && isFeatureEnabled('ember-no-double-extend')) {
+    expect(4);
```

Comment:
yeah `ignoreDeprecation` would allow you to not have this change in expect count for an unrelated test.